benchmarks/load_test.c

```diff
@@ -1,7 +1,29 @@
 /* This is a simple yet efficient WebSocket server benchmark much like WRK */
 
 #define _BSD_SOURCE
+
+#ifdef __APPLE__
+#include <libkern/OSByteOrder.h>
+
+#define htobe16(x) OSSwapHostToBigInt16(x)
```

Comment:
rationale for this was to make `load_test` work on macOS

macOS is missing these functions

Reply:
already included